composer/cli/launcher.py

```diff
@@ -185,14 +194,14 @@ def cleanup_processes(processes: Set[subprocess.Popen]):
     for process in processes:
         process.poll()
         if process.returncode is None:
-            print(f"Killing subprocess {process.pid} with SIGTERM")
+            log.info("Killing subprocess %s with SIGTERM", process.pid)
             try:
                 os.killpg(process.pid, signal.SIGTERM)
             except ProcessLookupError:
                 pass
 
     current_time = datetime.datetime.now()
-    print(f"Waiting {CLEANUP_TIMEOUT.seconds} seconds for processes to terminate...")
+    print(f"Waiting up to {CLEANUP_TIMEOUT.seconds} seconds for all training processes to terminate...")
```

Comment:
I'm still not sure what the cause of your crusade against f-strings is, but you missed one here :wink:

Reply:
`log` calls should not use f-strings because they are lazily constructed. But it's fine for print statements.